service/src/main/java/uk/gov/hmcts/reform/fpl/service/CreateC21OrderService.java

```diff
@@ -0,0 +1,95 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.google.common.collect.ImmutableMap;
+import org.assertj.core.util.Lists;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.C21Order;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Child;
+import uk.gov.hmcts.reform.fpl.model.common.C21OrderBundle;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+import uk.gov.hmcts.reform.fpl.utils.JudgeAndLegalAdvisorHelper;
+
+import java.time.format.FormatStyle;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+
+@Service
+public class CreateC21OrderService {
+
+    private final DateFormatterService dateFormatterService;
+    private final HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration;
+    private final Time time;
+
+    public CreateC21OrderService(DateFormatterService dateFormatterService,
+                                 HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration, Time time) {
```

Comment:
Time spaced down on to new line please. I am aprehensive about the use of Time however 👀